### PR TITLE
Ported DNX to NetCore RC2

### PR DESCRIPTION
**Pull request by hovsepm**

### CODE REVIEW

File: src/Search/Microsoft.Azure.Search/project.json [hunk 1]
```diff
@@ -1,57 +1,84 @@
 {
-  "version": "2.0.0-preview",
+  "version": "3.0.0-preview",
```
Comment (brjohnstmsft) [inline src/Search/Microsoft.Azure.Search/project.json]: Please do not increment the major version number for Search. We have not shipped any 2.x versions to NuGet yet. Also please do not publish this package as it is not ready to ship yet. We are currently doing feature development in another branch (search-v2-preview-dev).

File: src/Search/Search.Management.Tests/project.json [hunk 2]
```diff
@@ -1,33 +1,39 @@
...
   "authors": [ "Microsoft Corporation" ],
-  "tags": [ "" ],
-  "projectUrl": "",
-  "licenseUrl": "",
-  "commands": { "test": "xunit.runner.dnx" },
-  "compilationOptions": {
+
+  "packOptions": {
+    "summary": "Generator.CSharp Tests.",
+    "tags": [ "" ],
```
Comment (brjohnstmsft) [inline src/Search/Search.Management.Tests/project.json]: Incorrect package summary. This is a pseudo-MAML used only for testing the data plane Search SDK.

File: src/Search/Search.Tests/project.json [hunk 3]
```diff
@@ -2,30 +2,35 @@
...
-  "tags": [ "" ],
-  "projectUrl": "",
-  "licenseUrl": "",
-  "commands": { "test": "xunit.runner.dnx" },
-  "compilationOptions": {
+
+
+  "packOptions": {
+    "summary": "Generator.CSharp Tests.",
+    "tags": [ "" ],
```
Comment (brjohnstmsft) [inline src/Search/Search.Tests/project.json]: Incorrect summary. These are tests for the Search SDK.

File: README.md [hunk 4]
```diff
@@ -6,16 +6,13 @@ that take advantage of scalable cloud computing resources.
...
-    * dnxcore50, based on the .NET Core framework 
-    * .NET Portable Framework, using profile 111
+    * netstandard1.5, based on the NetCore framework 
+    * .NET Portable Framework/netstandard1.1 for NetCore, using profile 111
 
 ### Prerequisites:
-  Visual Studio 2015 RTM with ASP.NET. For details, check out the [installation doc](http://docs.asp.net/en/latest/getting-started/installing-on-windows.html). 
-  
-  Note, after done, run "_dnvm list_" command to check the 'coreclr' runtime is installed with right version of '_1.0.0-rc1-final_'. If not, run “_dnvm install 1.0.0-rc1-final -r coreclr -a x64 -Persistent_”. Remember always use "_-Persistent_" flag, so the selection can persist. 
+  Install CoreCLR RC2 using [these steps](https://www.microsoft.com/net/core#windows).
```
Comment (EmmaZhu) [inline README.md]: Minor: This should not only for windows.

**Comment by hovsepm** [comment]
@EmmaZhu , sorry merged before I noticed your comments. I'll update comments directly from github.

**Pull request merged**